packages/cross_file/CHANGELOG.md

```diff
@@ -1,6 +1,6 @@
-## NEXT
+## 0.3.4+2
 
-* Updates minimum supported SDK version to Flutter 3.16/Dart 3.2.
+* Supports `web: ">=0.5.1 <2.0.0"`.
```

Comment:
FWIW I would say "Adds support for `web` 1.x`." for all of these, rather than putting the full constraint details in the CHANGELOG.

Reply:
I kinda like the full constraint...just makes it SUPER clear what's going on. But 🤷

Reply:
Ret-conned the full constraint into ^1.0.0 (in practice I think it's easier to understand, maybe?)